data/lax_tap/mapping.csv

```diff
@@ -6,7 +6,6 @@ reader_id,stop_code,stop_id
 14,80205S,
 15,80201S,
 16,80212S,
-17,80214S,
```

Comment:
Stops on the Red Line go in order, from 11 to 25, and the pattern seemed to indicate that Union Station is stop 17, but when I actually went, it had the ID of 7, so I removed that ID from the list.